htdocs/install/mysql/data/llx_c_asset_disposal_type.sql

```diff
@@ -12,6 +12,14 @@
 --
 -- You should have received a copy of the GNU General Public License
 -- along with this program. If not, see <https://www.gnu.org/licenses/>.
+--
+--
 
+--
+-- Do not include comments at end of line, this file is parsed during install and string '--' are removed.
+--
 
-ALTER TABLE llx_asset_type_extrafields ADD INDEX idx_asset_type_extrafields (fk_object);
+INSERT INTO `llx_c_asset_disposal_type` (`rowid`, `entity`, `code`, `label`, `active`) VALUES
+(1, 1, 'C', 'Sale', 1),
```

Comment:
To be compatible with the error management system if install and migration, you must not used the "mysql compact version of sql" but 3 full real sql requests. So split this line in 3 lines.

Also, we must not include the rowid into the insert (not compatible with multicompany for example). The system will use the automatic increment system. We did it in past for some dictionary objects and it is still a problem. So can you instead do:
INSERT INTO llx_c_asset_disposal_type (entity, code, label, active) VALUES (1, 'C', 'Sale', 1);
INSERT INTO llx_c_asset_disposal_type (entity, code, label, active) VALUES...;
INSERT INTO llx_c_asset_disposal_type (entity, code, label, active) VALUES...;